website/docs/r/data_factory_linked_service.html.markdown

```diff
@@ -0,0 +1,102 @@
+---
+subcategory: "Data Factory"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_data_factory_linked_service"
+description: |-
+  Manages a Linked Service (connection) between a resource and Azure Data Factory.
+---
+
+# azurerm_data_factory_linked_service
+
+Manages a Linked Service (connection) between a resource and Azure Data Factory.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "example-resources"
+  location = "West Europe"
+}
+
+resource "azurerm_data_factory" "example" {
+  name                = "example"
+  location            = azurerm_resource_group.example.location
+  resource_group_name = azurerm_resource_group.example.name
+
+  identity {
+    type = "SystemAssigned"
+  }
+}
+
+resource "azurerm_storage_account" "example" {
+  name                     = "example"
+  resource_group_name      = azurerm_resource_group.example.name
+  location                 = azurerm_resource_group.example.location
+  account_kind             = "BlobStorage"
+  account_tier             = "Standard"
+  account_replication_type = "LRS"
+}
+
+resource "azurerm_data_factory_linked_service" "example" {
+  name            = "example"
+  data_factory_id = azurerm_data_factory.example.id
+  property_json   = <<JSON
+{
+  "annotations": [
+    "test1",
+    "test2",
+    "test3"
+  ],
+  "description": "test description",
+  "parameters": {
+    "foo": {
+      "type": "String",
+      "defaultValue": "test1"
+    },
+    "bar": {
+      "type": "Bool",
+      "defaultValue": true
+    }
+  },
+  "type": "AzureBlobStorage",
+  "typeProperties": {
+    "connectionString": "${azurerm_storage_account.example.primary_connection_string}"
+  }
+}
+JSON
+}
```

Comment:
can we please bring these shared propeties out into the resource instead of making this copmlicated json block?
```suggestion
  annotations = [t1,t2,t3]
  description = "a thing"
  type = "AzureBlobStorage"
  parameters = {}
  properties = {}
}
```

this allows us to do some validation and provider a much much better ux